client/src/pages/about-us/aboutUs.style.js

```diff
@@ -37,6 +37,24 @@ export const IntroductionDescription = styled.div`
   color: #000000;
 `;
 
+export const BrowseContainer = styled.div`
+  width: 650px;
+  height: 150px;
+  font-family: 'Montserrat', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  text-align: center;
+  margin: auto;
+  color: #2F7DA1;
+`;
+
+export const BorderContainer = styled.div`
+  width: 600px;
```

Comment:
<img width="957" alt="codeReviewss" src="https://user-images.githubusercontent.com/91857946/191126366-f674b5aa-0680-4f53-8b8d-380ec85583a9.png">
I saw this code's project through the front end, and I was able to see this red-to-blue gradient border. I was wondering if there's specific width for this border specified, or is it supposed to be good with the current width? Just pointing this out since it looked kinda awkward on the about-us front-end page.